index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/port.css">
+  <title>Patrick Portfolio</title>
+</head>
+<body>
+  <!--header-->
+  <div class="container">
+    <header class="header">
+      <nav class="navbar">
+        <a href="#" class="logo">
+          <img src="./images/Portfolio_Logo.png" alt="My Logo"/></a>
+         <a>
+          <img src="./images/Enabled.png" alt="Menu"/>
+        </a>
+      </nav>
+    </header>
+  <!--main section--> 
+    <section class="main_section">
+      <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+
+      <div>
+        <p class="media_p">LET'S CONNECT</p>
```

Comment:
- [ ] Kindly change this into a button or a link as stated in the project requierments

Reply:
Hi Aime, thanks for the review but I clearly don't understand your comments i look forward to your response